llvm/test/Transforms/LoopVectorize/pr72969.ll

```diff
@@ -0,0 +1,26 @@
+; RUN: not --crash opt -passes=loop-vectorize -S < %s
+
+target triple = "x86_64-unknown-linux-gnu"
```

Comment:
If this depends on the X86 backend, it needs to go in the `X86` subdirectory so it only runs when the X86 backend is built. Can it be made target-independent by using `-force-vector-width`/`-force-vector-interleave`?

Reply:
Ah right, I don't think it's necessary to duplicate the test, having it X86 only is fine.